Documentation/config/core.txt

```diff
@@ -66,7 +66,11 @@ core.fsmonitor::
 	will identify all files that may have changed since the
 	requested date/time. This information is used to speed up git by
 	avoiding unnecessary processing of files that have not changed.
-	See the "fsmonitor-watchman" section of linkgit:githooks[5].
++
+See the "fsmonitor-watchman" section of linkgit:githooks[5].
++
+Note: this option is ignored if the built-in FSMonitor is enabled via the
+`core.useBuiltinFSMonitor` option (see linkgit:git-fsmonitor--daemon[1]).
 
 core.fsmonitorHookVersion::
 	Sets the version of hook that is to be used when calling fsmonitor.
```

Comment:
?? ... when calling the fsmonitor command specified in `core.fsmonitor` ??

I know I'm being paranoid here.  I'm just looking for a way to tie `core.fsmonitor` and `core.fsmonitorHookVersion` together and break any association with our new builtin setting.

Reply:
Good point!